projects/account/src/app/modules/device/components/card/software-release-card/software-release-card.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { OptionButtonsConfig } from '@account/models/option-buttons-config.model';
+
+@Component({
+    selector: 'account-software-release-card',
+    templateUrl: './software-release-card.component.html',
+    styleUrls: ['./software-release-card.component.scss']
+})
+export class SoftwareReleaseCardComponent implements OnInit {
+    @Input() softwareReleaseForm: FormGroup;
+    public releaseChannelConfig: OptionButtonsConfig;
+
+    constructor() {
+        this.releaseChannelConfig = {
+            options: ['Stable', 'Latest'],
```

Comment:
In the future we could ping the API to show all channels available for a given device ID, but makes sense to have them hard coded for the moment.

Reply:
That could be a lot of pings for an account with a lot of devices.  Could just ping the API that lists all the possible channels for the fleet. But I agree that this should come from the API eventually